server/settings/tools.py

```diff
@@ -50,9 +50,11 @@ class FilterCreatorProfile(BaseSettingsModel):
     task_names: list[str] = SettingsField(
         default_factory=list,
         title="Task names")
-    creator_identifiers: list[str] = SettingsField(
+    creator_labels: list[str] = SettingsField(
         default_factory=list,
-        title="Allowed Creator Identifiers")
+        title="Allowed Creator Labels",
+        description="Copy creator label from Publisher, regex supported."
+    )
```

Comment:
Oh boy - is matching by label really the best choice here? I understand `identifiers` aren't necessarily easily accessible or whatever but at least they are supposed to be static. But labels feel way more 'fluid' than that.

How about we just allow to enable creators based on their `product_type`? Which may be even easier than the label, and at least also considered stable? (We might still need the additional filtering to 'identifier' for the specific cases where multiple creators exist for a single product type - which may happen in some cases).

Reply:
That doesn't match planned aliasing of creators. If I understood it correctly, you might have `ImageHD`, `ImageLD` creators both producing `image` product type, but you would like still to show only `ImageHD` for one task and the other creator for different.

Reply:
I see. So 'aliases' would still have the same identifier as well, just different labels?

Anyway, I might be nitpicky. But "labels" definitely don't seem like a constant 'name' to me we should be using as an identifier or key for a particular creator but maybe this use case warrants it.

Reply:
You can't have create 2 plugins with same identifier.

Reply:
Yeah, using labels as keys sounds really bad, but I don't think any lead would like to dive into code to get those, product type seems reasonable too, but you would still figure out if that product type is actually produced in DCC, with Labels it seems pretty straightforward to open DCC and Publisher, copy&paste from it directly into Settings with no fuss.

I dont think we are changing labels too often, they are pretty static after they are implemented, imho. It could bite us if we would like to do internationalization, but that would open completely new can of worms so this one would be the smallest issue.